proto/regen/group/v1alpha1/query.proto

```diff
@@ -0,0 +1,244 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+import "regen/group/v1alpha1/types.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+// Query is the regen.group.v1alpha1 Query service.
+service Query {
+
+  // GroupInfo queries group info based on group id.
+  rpc GroupInfo(QueryGroupInfoRequest) returns (QueryGroupInfoResponse);
+
+  // GroupAccountInfo queries group account info based on group account address.
+  rpc GroupAccountInfo(QueryGroupAccountInfoRequest) returns (QueryGroupAccountInfoResponse);
+  
+  // GroupMembers queries members of a group
+  rpc GroupMembers(QueryGroupMembersRequest) returns (QueryGroupMembersResponse);
+
+  // GroupsByAdmin queries groups by admin address.
+  rpc GroupsByAdmin(QueryGroupsByAdminRequest) returns (QueryGroupsByAdminResponse);
+
+  // GroupAccountsByGroup queries group accounts by group id.
+  rpc GroupAccountsByGroup(QueryGroupAccountsByGroupRequest) returns (QueryGroupAccountsByGroupResponse);
```

Comment:
IMHO, would be better to remove Group prefix to avoid name clashes:

```
groupClient.GroupAccountsByGroup
```

Reply:
we can deal with that later... can you open an issue or PR if you think there's something that can be improved

Reply:
@blushi  - do you have an opinion about naming? If you also find that we could improve naming then I'm happy to create a PR.

Reply:
@robert-zaremba you mean having `AccountsByGroup` instead for instance? I don't have a strong preference, although I think keeping the Group prefix makes it more explicit.

Reply:
> I think keeping the Group prefix makes it more explicit.

This context is already provided by a client / server object, right?
```
groupClient.AccountsByGroup
vs
groupClient.GroupAccountsByGroup   // we mention group 3 times, and it also sounds that we are grouping by (like in SQL)
```